Neos.Eel/Tests/Unit/Helper/DateHelperTest.php

```diff
@@ -124,6 +124,72 @@ public function formatCldrCallsFormatService()
         $helper->formatCldr($date, $format, $locale);
     }
 
+    /**
+     * @test
+     */
+    public function formatCldrDate()
+    {
+        $date = \DateTime::createFromFormat('Y-m-d H:i:s', '2013-07-03 12:34:56');
+        $formatLength = 'long';
+        $localeString = 'en';
+
+        $locale = new Locale('en');
+        $expectedString = '2013-07-03 12:34:56';
+
+        $formatMock = $this->createMock(\Neos\Flow\I18n\Formatter\DatetimeFormatter::class);
+        $formatMock->expects(self::atLeastOnce())->method('formatDate')->with($date, $locale, $formatLength)->willReturn($expectedString);
+
+        $helper = new DateHelper();
+        $this->inject($helper, 'datetimeFormatter', $formatMock);
+
+        $result = $helper->formatCldrDate($date, $formatLength, $localeString);
+        $this->assertEquals($expectedString, $result);
+    }
+
+    /**
+     * @test
+     */
+    public function formatCldrTime()
+    {
+        $date = \DateTime::createFromFormat('Y-m-d H:i:s', '2013-07-03 12:34:56');
+        $formatLength = 'long';
+        $localeString = 'en';
+
+        $locale = new Locale('en');
+        $expectedString = '2013-07-03 12:34:56';
```

Comment:
Shouldn't the expected string be just the time and in the other test just the date, all three methods now have the same output in the test?

Reply:
Ah now I get it, the mock is just copied.

Reply:
Yeah, it doesn't matter as the test, simply checks if the params are passed properly to the service and if the result will be returned.